### DISCUSSION TITLE
TSM Questions / Suggestion

### DISCUSSION THREAD

**Discussion by rebop** [sienori/Tab-Session-Manager#782] [Q&A]
I am still looking for a Session Manager I like as much as the "original" Session Manager. TSM does come the closest.

**Issue**: When opening a tab group (session) the focus is on the last tab of the group. I would like it to be on the first tab. Can that be changed? I am a bit of a geek so happy to open the hood and tweak if that can be done.

**Question**: Where are these stored? Simply cannot find them. I keep multiple computers in sync by copying the "Session Manager" Session folder so I always am in sync now, but cannot find where these are stored. Nothing at all in browser-extension data.

**Suggestion**: Would be nice to change the highlight color. Not the theme, just the blue to my choice to better match my overall themes. Small stuff.

**Bigger stuff suggestion:** I **REALLY** miss having a popup open when opening the browser that asks which Session to open or none. So handy in Session Manager.

Thanks!

~Bob

**Comment by sienori** [comment]
> Issue: When opening a tab group (session) the focus is on the last tab of the group. I would like it to be on the first tab. Can that be changed? I am a bit of a geek so happy to open the hood and tweak if that can be done.

When restoring a session, the last focused tab will be focused. Currently, to change the focused tab, you need to open a session, change the tab focus, and save it. I plan to implement the ability to edit the focused tab in the future.

> Question: Where are these stored? Simply cannot find them. I keep multiple computers in sync by copying the "Session Manager" Session folder so I always am in sync now, but cannot find where these are stored. Nothing at all in browser-extension data.

See: https://github.com/sienori/Tab-Session-Manager/wiki/Q&A:-Sessions-data-storage-location;-How-to-migrate-sessions-data-to-another-profile

> Suggestion: Would be nice to change the highlight color. Not the theme, just the blue to my choice to better match my overall themes. Small stuff.

The Tree Style Tab provides a mechanism for users to set CSS. It might be a good idea to implement such a mechanism...

> Bigger stuff suggestion: I REALLY miss having a popup open when opening the browser that asks which Session to open or none. So handy in Session Manager.

Good idea! If there are many requests, I may implement it. Can you create an individual issue?

**Comment by rebop** [comment]
Thanks so much for the reply!

As to "last focused tab", this happens when I import sessions. The imported session in its native app DOES open on first tab of the group. TSM always opens to the last. I will try your suggestion.

Sorry, I am not finding the .sqlite file anywhere in my profile. Nor a TSM folder. 

I am familiar with CSS. I have a userchrome.css and a usercontent.css. Unsure what to try to set to change the color on TSM only.

I shall post the suggestion.

Thanks again.

**Comment by rebop** [comment]
Changing the session tab focus and saving works! Now would be nice if I could write over the session when I save. Perhaps I am nos seeing that option? Now I name the session same as the one opened and save. then delete the original session.

**Comment by rebop** [comment]
Ooops, typed too soon. Replace with current session does it perfectly. This will work for me. Thanks.

**Comment by rebop** [comment]
I have tried to follow this:

See: https://github.com/sienori/Tab-Session-Manager/wiki/Q&A:-Sessions-data-storage-location;-How-to-migrate-sessions-data-to-another-profile

However cannot find de50ed3a-7f77-40bc-af2e-c5953d30edbb anywhere. Can you be a bit more specific where this is found? Thanks.

**Reply by sienori** [reply]
The folder name `de50ed3a-7f77-40bc-af2e-c5953d30edbb` will vary depending on the environment.
This is the Internal UUID of TSM. You can find it at `about:debugging`.
![](https://raw.githubusercontent.com/wiki/sienori/Tab-Session-Manager/img/aboutDebugging1.png)
![](https://raw.githubusercontent.com/wiki/sienori/Tab-Session-Manager/img/aboutDebugging2.png)

**Comment by rebop** [comment]
Got it. Thanks. Difficult place to find, but....
